src/index.es.ts

```diff
@@ -10,18 +9,79 @@ import {
   getInsightsAnonymousUserToken,
 } from './helpers';
 import { createInfiniteHitsSessionStorageCache } from './lib/infiniteHitsCache';
+import { deprecate } from './lib/utils';
 
-const instantsearch = (options: InstantSearchOptions): InstantSearch =>
+interface InstantSearchModule {
```

Comment:
Any reason you went for `interface` instead of `type` here?

Reply:
initially I thought callable + properties was only possible on interface, but it looks like type is possible too. Let's go with type for consistency